Lesson Plans/Private/XC.9 Emergency Operations.md

```diff
@@ -23,42 +23,107 @@ Develop knowledge of all abnormal and emergency procedures for the training airc
 > Can be completed at the same time as [[XC.8 Emergency Flight by Instruments - Four Fundamentals]] or [[XC.10 Lost Procedures and Diversion to Alternates]].
 
 ### Lesson Elements
+##### Handling Emergencies
 1. [[FAR 91.3 PIC Responsibility and Authority]]
-3. What is an emergency? When should you declare one?
+2. What is an emergency? When should you declare one?
 	1. When the outcome of the flight is in doubt
-4. Instrument failures (vacuum or pitot-static system)
-5. Fire
-	1. Engine fire
-		1. In air - Emergency Descent (this is the rapid descent procedure)
-			1. Start descent (bank 30°-45°; airspeed >=100 kts; should have positive load factor)
-			2. Then once fire out, execute forced landing (see [[SOLO.15 Emergency Landings]])
-		2. During start on ground
-	2. Electrical Fire
-	3. Wing Tip Fire
-6. Failure
-	1. Engine
-		1. During flight: see [[SOLO.15 Emergency Landings]]
-		2. During takeoff roll
-		3. Immediately after takeoff: Land straight ahead!^[[Cub Theatre: The-Not-That-Impossible Turn](https://www.youtube.com/watch?v=byBh1Qd6IgA) from [[Paul Bertorelli]] of [[AVweb]]]
-	2. Electrical
-		1. Malfunctions or complete failure
-		2. Nordo (No Radio)
-	3. Minimum fuel
-	4. Flap failure
-	5. Retractable gear malfunction
-	6. Door opening in flight
-	7. Low oil pressure
-7. Unsafe Conditions
-	1. [[VFR]] into [[IMC]]
-	2. Icing
-		1. [[How and Why Carburetor Ice Forms]]
-8. Emergency equipment and survival gear
-9. Emergency squawk codes: [[Squawk 7700]], [[Squawk 7600]], [[Squawk 7500]]
+	2. When you need higher priority handling from ATC
+
+##### Something is Wrong!
```

Comment:
I also emphasize the importance of understanding the aircraft systems, since emergencies are often weirder than this. Like I smelled burning insulation flying over KPAO once. My student and I ran the electrical fire procedure, but I quickly radioed the tower to tell them we needed to turn off the electrical and expected to be back soon, or need to land NORDO. He said fine. We ran the procedure, and turned on just radio to land. Then on the ground we tried each system and could reproduce the issue. We finally decided it was just a burning something coming from East Palo Alto, and not actually from our plane at all.

This was a good example of premeditating an emergency (thinking to call tower quickly before just shutting things off), CRM (I took running checklist and radio, student focused on flying and landing), and value of knowing the systems (knowing which things I can turn on and what can stay off, to minimize likelihood of restarting whatever the fire was).

Reply:
I guess for the actual lesson plans, could just mention handling a real emergency requires you to have practiced them so you have the brain cycles available to think about the other things going on. Or something more profound then that... I'm not sure, I feel like I'm rambling now so stopping this comment here. Do what you'd like with it :D

Reply:
These are great scenarios to discuss.  I won't add them to the outline, but I'll put them in my notes for discussion with students.  Thanks.